doc/examples/placement-group.rst

```diff
@@ -28,9 +28,10 @@ Gang Scheduling
 ---------------
 **Recommended Strategy**: `STRICT_SPREAD`.
 
-Sometimes, you'd like to schedule multiple tasks/actors in a separate physical machine (node) "at the same time". For example, "write the gang scheduling example".
+Sometimes, you'd like to schedule multiple tasks/actors in a separate physical machine (node) "at the same time".
```

Comment:
@richardliaw Do you mind giving me an example of gang scheduling in ML related workload? I don't have good enough knowledge to write that here.

Reply:
Nice catch -- 
```
For example, you may use a collective communication library like NCCL and form a communication group with a set of actors. In creating this set of actors, you may want to ensure that all actors are separated and placed evenly across available nodes (creating a homogenous cluster).